lecture2gether-vue/src/main.ts

```diff
@@ -3,13 +3,34 @@ import App from './App.vue';
 import router from './plugins/router';
 import store from './plugins/store';
 import vuetify from './plugins/vuetify';
+import * as Sentry from "@sentry/browser";
+import {Vue as VueIntegration} from "@sentry/integrations/dist/vue";
 
 
 Vue.config.productionTip = false;
 
-new Vue({
-    router,
-    store,
-    vuetify,
-    render: (h) => h(App),
-}).$mount('#app');
+// Fetch settings because we can only initialize sentry before Vue      TODO Improve because loading delay
```

Comment:
Whats up with this todo? Is it necessary?

Reply:
The todo signals that this should be improved but I don't know how so that's why it's still there.